src/views/layouts/main.hbs

```diff
@@ -4,12 +4,14 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="/style.css">
-    <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
```

Comment:
Is there any reason you've changde it  back to 4.7.0??, we had some weird errors and bugs with the tachyons version before that's why I've changed it

Reply:
This PR was opened 23 days ago i totally forgot why i changed it.
i think it was something to do with tachyons's media queries

Reply:
I would approve this since you don't remember but if it got merged we need to check that we don't get any errors from tachyons